.github/fabricbot.json

```diff
@@ -1585,7 +1585,8 @@
         "fabiocav",
         "brettsam",
         "satvu",
-        "kshyju"
+        "kshyju",
+        "jviau"
       ],
       "modifiedBy": {
         "upn": "facaval@microsoft.com",
```

Comment:
Should we also update the modifiedBy data?

Reply:
I don't think it's relevant xD